test/multilib/multilib_success_commands.test

```diff
@@ -0,0 +1,1936 @@
+; Success
+
```

Comment:
Can this test be split into multiple files? It'd help with maintainability and allow things to be tested in parallel.
A suggestion for the initial split would be something like:
- aarch64_a_supported_variants.test
- aarch64_r_supported_variants.test
- arm_a_supported_variants.test
- arm_r_supported_variants.test
- arm_m_supported_variants.test